subsystems/indexersubsystem.py

```diff
@@ -131,14 +131,18 @@ def periodic(self) -> None:
                 constants.kStagingSpeed * constants.kTalonVelocityPerRPM,
             )
         elif self.state == self.Mode.Holding:
-            self.indexerMotor.set(
-                ControlMode.Velocity,
-                constants.kIndexerSpeed * constants.kTalonVelocityPerRPM,
-            )
-            self.stagingMotor.set(
-                ControlMode.Velocity,
-                -constants.kStagingSpeed * constants.kTalonVelocityPerRPM,
-            )
+            if not (self.indexerSensor() == 0 and self.stagingSensor() == 0):
```

Comment:
Why not if (resolves true)? Is there a reason this is specifically if NOT instead?

Reply:
Can shorten to `if (self.indexerSensor() != 0 or self.stagingSensor() != 0)`

Reply:
This probably doesn't really make a performance difference so I'm gonna skip over it for now